rosdep/base.yaml

```diff
@@ -2896,6 +2903,14 @@ log4cxx:
   opensuse: [liblog4cxx-devel]
   rhel: [log4cxx-devel]
   ubuntu: [liblog4cxx10-dev]
+lttng-modules:
+  arch: [lttng-modules]
+  debian: [lttng-modules-dkms]
+  fedora: [lttng-devel]
```

Comment:
I think this rule might be invalid as on Fedora it says you need to compile the the modules from source: https://lttng.org/download/#fedora